bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/bstar_tW_Left_CASE/bstar_tW_MB_MW/bstar_tW_MB_MW_proc_card.dat

```diff
@@ -0,0 +1,18 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model Bprime_no_b_mass
+define p = g u c d s b u~ c~ d~ s~ b~
+define j = g u c d s b u~ c~ d~ s~ b~
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+add process p p > t w- QAD=2 QCD=0 QED=0
+add process p p > t~ w+ QAD=2 QED=0 QCD=0
+add process p p > t~ w+ QAD=2 QCD=0 QED=0
+add process p p > t w- QAD=2 QED=0 QCD=0
+add process p p > t~ w+ QAD=2 QED=0 QCD=0
+add process p p > t w- QAD=2 QED=0 QED=0
+add process p p > t~ w+ QAD=2 QED=0 QCD=0
```

Comment:
you need firstly "generate p p > t w- ...." and then add process. and there are duplicate process here, you should only have two processes, t w-, t~ w+

Reply:
Hi @menglu21 , yep that's a silly mistake.  Originally each of these lines did a different decay mode, but we're going to do the decays with pythia I believe.  I added in the generate line